website/docs/addon_maya_ornatrix_artist.md

```diff
@@ -98,3 +68,19 @@ The cache family exports Ornatrix Alembic (.abc) which includes the Ornatrix dat
 ### Loading
 
 The **Ornatrix Cache** supports to load Ornatrix alembic (.abc), Vray Proxy (.abc) and Arnold Scene Source (.abc).
+
+## FAQ
+### How to connect an Ornatrix rig to a pointcache or animation?
+
+To guide Ornatrix rig to its proper location. you'll need to connect it to the animation, aligning both transform and mesh attributes.
```

Comment:
```suggestion
To position the Ornatrix rig correctly, you need to connect it to the animation, aligning both the transform and mesh attributes
```

Reply:
fixed in https://github.com/ynput/ayon-documentation/pull/237/commits/7a84098e9fe5327cfc62278b0b720399bf206088